README.md

```diff
@@ -1,5 +1,7 @@
 # (Poly)merase &middot; [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)  [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebook/react/blob/master/LICENSE) ![Build Status](https://travis-ci.org/TimothyStiles/poly.svg?branch=master) ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg) 
-Poly is a Go library and command line utility for engineering organisms.
+Poly is a Go library and command line utility (CLI) for engineering organisms.
```

Comment:
I'd remove this bit. CLI doesn't make sense as an acronym coming after command line utility.